subgraph.yaml

```diff
@@ -659,9 +659,9 @@ dataSources:
         - SaleAuction
       eventHandlers:
         - event: File(indexed bytes32,uint256)
-          handler: handleFile
+          handler: handleFile1
```

Comment:
Maybe change the name to handleFileBigInt

Reply:
I want to keep the default function handlers.